nix/crossBuild.nix

```diff
@@ -0,0 +1,85 @@
+{ nixpkgs, system, version }:
+let
+  # ld flag to embed the version in the binary
+  ldEmbedVersion =
+    "-X github.com/dividat/driver/src/dividat-driver/server.version=${version}";
+
+  buildGoModuleFor = targetSystem: customAttrs:
+    let
+      crossPkgs = (import nixpkgs {
+        system = system;
+        crossSystem.config = targetSystem;
+      });
+    in
+    with crossPkgs;
+    buildGoModule ({
+      pname = "dividat-driver";
+      version = version;
+      vendorHash = "sha256-S8qJorySFPERNp4i9ckeZCPLyx0d/QweS4hm4ghJT4k=";
```

Comment:
Leave short note on how to update this?

Reply:
This was made redundant by [Provide cross-building toolchain in nix shells](https://github.com/dividat/driver/pull/129/commits/f8907ae140d171ca4444585e862ff1557fe1a9be)